### DISCUSSION TITLE
RabbitMQ Managment not showing correctly

### DISCUSSION THREAD

**Discussion by dvRox84** [rabbitmq/rabbitmq-server#12645] [Questions] [answered]
### Describe the bug

RabbitMQ Managment does not show entire page. Cannot find 'Overview', 'Queues' etc tabs.

### Reproduction steps

I have installed rabbitMQ on Windows Server 2016. Everything else seems to work fine except this.

### Expected behavior

I expect to see full interface

### Additional context

Windows

**Comment by lukebakken** [comment]
Please read Team RabbitMQ's community support policy:

https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md#who-is-eligible-for-community-support

In your case, you don't even bother to specify which version of RabbitMQ and Erlang you are using, nor which web browser. How do you expect to get assistance? My guess is that the browser you are using on Win 2016 is too old to display the management interface.

**Comment by michaelklishin** [comment, marked as answer]
@dvRox84 [clear your browser's cache](https://www.rabbitmq.com/docs/upgrade#rabbitmq-plugins-compatibility) and refresh the page.

And perhaps give enough of a damn to provide version information next time you want to ask a question.
